go.mod

```diff
@@ -66,7 +66,7 @@ require (
 	github.com/basgys/goxml2json v1.1.0
 	github.com/bxcodec/faker v2.0.1+incompatible
 	github.com/cloudquery/cq-gen v0.0.5
-	github.com/cloudquery/cq-provider-sdk v0.10.6
+	github.com/cloudquery/cq-provider-sdk v0.10.7
```

Comment:
I'm confused with this change as `main` already references `v0.10.7`
https://github.com/cloudquery/cq-provider-aws/blob/713a1d9110b4bb686fca28961c7ff36a7bad9e18/go.mod#L69